src/App.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="tile-container">
+    <BackgroundTile type="image" src="https://hdwallsource.com/img/2014/6/mountain-peaks-wallpaper-hd-33593-34349-hd-wallpapers.jpg" />
```

Comment:
Could you move this Component outside of `.tile-container`? Also, it would be nice if type and src would be from `App` data